templates/web/legacy/embedded_legacy_form.ftl

```diff
@@ -91,6 +91,7 @@
     const newEmbedded = CStudioAuthoring.Utils.getQueryVariable(location.search, 'newEmbedded');
     const contentTypeId = CStudioAuthoring.Utils.getQueryVariable(location.search, 'contentTypeId');
     const isNewContent = CStudioAuthoring.Utils.getQueryVariable(location.search, 'isNewContent') === 'true';
+    const formId = CStudioAuthoring.Utils.getQueryVariable(location.search, 'formId');
```

Comment:
Use id will likely have characters like `:`; URL-encode/decode where necessary here and when receiving the message from this frame.